tests/flytekit/unit/core/test_map_task.py

```diff
@@ -1,6 +1,7 @@
 import functools
 import typing
 from collections import OrderedDict
+from flytekit.core.resources import Resources
```

Comment:
```suggestion
from flytekit import Resources
```

Reply:
Can you also sort the imports using isort?

Reply:
Done👍